src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -228,7 +226,7 @@ public void testCleanupNonExistentDir() throws Exception {
         when(dirToDelete.exists()).thenReturn(false);
 
         final boolean result = Executrix.cleanupDirectory(dirToDelete);
-        assertThat(result, is(true));
+        assertTrue(result);
```

Comment:
Consider reverting these changes in favor of the more sweeping changes to this file in PR #230